api/notd/token_metadata_processor.py

```diff
@@ -91,7 +92,10 @@ async def retrieve_token_metadata(self, registryAddress: str,tokenId: str) -> Re
             tokenMetadataDict = {}
         elif tokenMetadataUri.startswith('data:'):
             # TODO(krishan711): parse the data here
-            tokenMetadataDict = {}
+            basestr = tokenMetadataUri.replace('data:application/json;base64', '')
```

Comment:
this hsould only be done if it starts with this whole prefix (there may be other `data:` encodings)

Reply:
base64Metadata would be a better variable name